docs/releasing.md

```diff
@@ -45,17 +47,17 @@ In the Heroku CLI, we have it automatically build and release the beta channel o
 
 ## Windows installer
 
-Build a windows installer with `oclif pack:win`. It will build into `./dist/win`. This can be published to S3 with `oclif promote --win`.
+Build a windows installer with `oclif pack:win`. It will build into `./dist/win`. This can be uploaded to S3 with `oclif upload:win` and promoted within S3 with `oclif promote --win`.
```

Comment:
replace `oclif pack:win` with `oclif pack win` `oclif upload:win` with `oclif upload win` I know ':' still works but use space for consistency.